src/aspire/utils/misc.py

```diff
@@ -73,11 +85,14 @@ def src_wiener_coords(sim, mean_vol, eig_vols, lambdas=None, noise_var=0, batch_
     """
 
     if not isinstance(mean_vol, Volume):
-        logger.warning('src_wiener_coords mean_vol should be a Volume instance. Correcting for now.')
+        logger.warning('src_wiener_coords mean_vol should be a Volume instance. Attempt correction.')
```

Comment:
`logger.debug`? (Judging from similar messages above.)

Reply:
Sure, I'm hoping we get away from hitting these, but for now maybe best to leverage `debug`. We are only ones need to know atm.